tui/fake_dev_test.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+bash build.sh
+./build/main build/FAKE_SERIAL_DEVICE.bin
+hexdump -C build/FAKE_SERIAL_DEVICE.bin
```

Comment:
I didn’t even realize that there’s a hexdump command. I would’ve done `./build/main /dev/stdout | xxd`